src/widgets/chart.rs

```diff
@@ -97,8 +101,8 @@ impl<'a> Axis<'a> {
     ///         .labels(vec!["0".bold(), "25".into(), "50".bold()]);
     /// ```
     #[must_use = "method moves the value of self and returns the modified value"]
-    pub fn labels(mut self, labels: Vec<Span<'a>>) -> Self {
-        self.labels = Some(labels);
+    pub fn labels<T: Into<Line<'a>>>(mut self, labels: Vec<T>) -> Self {
+        self.labels = Some(labels.into_iter().map(Into::into).collect());
```

Comment:
Why not `IntoIterator` then too? That would allow for `.labels(["0.0", "5.0", "10.0"])` without `vec![]`.

Reply:
Sounds good - want to slap together a quick PR for that?